build-support/src/main/kotlin/app/cash/redwood/buildsupport/RedwoodBuildPlugin.kt

```diff
@@ -523,7 +523,7 @@ private class RedwoodBuildExtensionImpl(private val project: Project) : RedwoodB
         // Note: This makes assumptions about our setup having a JS target with the default name.
         it.files.from(project.tasks.named("compileProductionExecutableKotlinJsZipline"))
         it.appName.set(name)
-        it.outputDirectory.set(project.layout.buildDirectory.dir("zipline"))
+        it.outputDirectory.set(project.layout.buildDirectory.dir("treehouse"))
```

Comment:
I took longer than I should have to realize our builds were failing due to a path name collision.